actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadManagementActor.java

```diff
@@ -214,6 +215,28 @@ private void getUploadStatus(Request actorMessage) {
                       addTaskDataToList(failureList, x.getFailureResult());
                     }
                   });
+          if (!(JsonKey.USER.equalsIgnoreCase(objectType))) {
```

Comment:
JsonKey.USER.equalsIgnoreCase(objectType) - why this check is not for user. it should be for user only. Please check